src/main/resources/application.yaml

```diff
@@ -190,7 +190,8 @@ scheduling:
       cron: ${DELETE_REJECTED_FILES_CRON}
       ttl: ${DELETE_REJECTED_FILES_TTL}
     delete-complete-files:
-      enabled: false
+      enabled: ${DELETE_COMPLETE_FILES_ENABLED:false}
```

Comment:
can we remove default ?

Reply:
having default value makes it bullet proof. other properties have default values. why do you suggest removing default?

Reply:
there are some conflicts in bootstrapping those application configs in during flux deployments on different environments. the working solution is to remove defaults